File: notification/android/app/src/main/java/com/example/notification/NotificationActions.java
package com.example.notification;

import android.app.Activity;
import android.app.NotificationChannel;
import android.app.NotificationManager;
import android.app.Service;
import android.content.Intent;
import android.os.Bundle;
import android.os.IBinder;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

//import io.flutter.embedding.android.FlutterView;
import io.flutter.embedding.android.FlutterActivity;
import io.flutter.embedding.engine.FlutterEngine;
import io.flutter.view.FlutterView;
import io.flutter.plugin.common.BinaryMessenger;
import io.flutter.plugin.common.MethodChannel;

public class NotificationActions extends FlutterActivity {
//    private FlutterView flutterView;
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        System.out.println("created");
    }

    public void configureFlutterEngine(@NonNull FlutterEngine flutterEngine) {
        super.configureFlutterEngine(flutterEngine);
        System.out.println("configure");
//        new MethodChannel(flutterEngine.getDartExecutor().getBinaryMessenger(), "com.example.notification.messages").invokeMethod("message","msd");
    }

    @Override
    protected void onStart() {
        super.onStart();
        System.out.println("strated");
//        FlutterEngine flutterEngine=new FlutterEngine();
//        new MethodChannel(flutterEngine.getDartExecutor().getBinaryMessenger()).invokeMethod("message","msd");
//        FlutterActivity flutterActivity= new FlutterActivity();
//        new MethodChannel(flutterActivity.getFlutterView(), "com.example.notification.messages1").invokeMethod("message",null);
    }
}

File: android/app/src/main/kotlin/com/example/notifkotlin/NotificationActions.kt
package com.example.notifkotlin
import android.app.Activity
import android.os.Bundle
import android.os.PersistableBundle
import io.flutter.app.FlutterActivity
import io.flutter.plugin.common.MethodChannel
import io.flutter.plugins.GeneratedPluginRegistrant

//import io.flutter.embedding.android.FlutterView;
//import io.flutter.embedding.android.FlutterActivity;
class NotificationActions : FlutterActivity() {
        override fun onStart() {
                super.onStart()
        }

        override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
                super.onCreate(savedInstanceState, persistentState)
        }
        override fun onCreate(savedInstanceState: Bundle) {
                super.onCreate(savedInstanceState)
//        GeneratedPluginRegistrant.registerWith(this)
//                val methodChannel = MethodChannel(flutterView, "com.example.notification.messages1")
//                methodChannel.invokeMethod("message", "leave_call", object : MethodChannel.Result {
//                        override fun success(result: Any?) {
//                                println(result)
//                        }
//
//                        override fun error(errorCode: String, errorMessage: String?, errorDetails: Any?) {
//                                println(errorCode)
//                        }
//
//                        override fun notImplemented() {
//                                println("Unrealized getPlatform Method")
//                        }
//                })
        }
}